### PR TITLE
feat(route): add 软餐

### LINKED ISSUE #8230: 希望支持软餐
### 网站地址
https://www.ruancan.com/
### 网站描述
软餐创建于2011年，是聚焦国内外移动应用、PC软件生态和业界消息的原创科技媒体。
### 需要生成什么内容？
首页全文 RSS 输出，或分 tag 输出
### 额外描述
网站自带的 RSS (https://www.ruancan.com/feed) 不支持全文输出，也无法按 tag 分类

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #8230

## 完整路由地址 / Example for the proposed route(s)

```routes
/ruancan
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/ruancan/radar.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    'ruancan.com': {
+        _name: '软餐',
+        '.': [
+            {
+                title: '首页',
+                docs: 'https://docs.rsshub.app/new-media.html#ruan-can-shou-ye',
+                source: ['/'],
+                target: '/',
```
Comment (TonyRL) [inline lib/v2/ruancan/radar.js:9]:
```suggestion
                target: '/ruancan',
```
[script-standard](https://docs.rsshub.app/joinus/script-standard.html) has a typo in radar rules.
[quick-start](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-radar-gui-ze) has the right format.

File: lib/v2/ruancan/radar.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            },
+            {
+                title: '分类',
+                docs: 'https://docs.rsshub.app/new-media.html#ruan-can-fen-lei',
+                source: ['/sort/:sort', '/'],
+                target: '/sort/:sort',
```
Comment (TonyRL) [inline lib/v2/ruancan/radar.js:15]:
```suggestion
                target: '/ruancan/sort/:sort',
```

File: lib/v2/ruancan/radar.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            },
+            {
+                title: '标签',
+                docs: 'https://docs.rsshub.app/new-media.html#ruan-can-biao-qian',
+                source: ['/tag/:tag', '/'],
+                target: '/tag/:tag',
```
Comment (TonyRL) [inline lib/v2/ruancan/radar.js:21]:
```suggestion
                target: '/ruancan/tag/:tag',
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/ruancan/radar.js (hunk 1 above)
Comment (nczitzk) [reply lib/v2/ruancan/radar.js:9]: thank you for your review ❤️

**Pull request merged**